### PR TITLE
block-scope: corrections. Fixes gh-201

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

**Comment by arv** [comment]
LGTM

File: test/language/block-scope/syntax/for-in/identifier-let-allowed-as-lefthandside-expression.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
-es6id: 13.1
+es6id: 13.6.0.1
 description: >
     for declaration:
-    missing identifier, "let" disallowed as bound name
-negative: SyntaxError
+    identifier "let" allowed as lefthandside expression
 ---*/
```
Comment (bterlson) [inline test/language/block-scope/syntax/for-in/identifier-let-allowed-as-lefthandside-expression.js]: I think you need "noStrict" for this one, though I think you could declare "var let" earlier in the test to make it work in strict mode too.

Comment (rwaldron) [inline test/language/block-scope/syntax/for-in/identifier-let-allowed-as-lefthandside-expression.js]: > I think you need "noStrict" for this one

Yep. Update coming. 


> I think you could declare "var let" 

Should be another test.

**Comment by bterlson** [comment]
Landed this.

**Pull request closed without merging**